### ISSUE TITLE
Text fragmentation and flicker

### ISSUE THREAD

**Issue by mahalde** [brave/brave-browser#3071]
## Description 

Whenever I open the browser, strange text fragmentation can be seen. It sometimes happens when I open the site, sometimes only after I interact with it. This isn't contained to website, also Brave intern sites like the settings or the DevTools are affected. Also on scrolling the text flickers into different fragmentations.

## Steps to Reproduce

   1. Open Brave Browser
   2. Watch any site and interact with it
   3. Wait and eventually the screen errors show up

## Actual result:

Brave rewards looks like this:
![brave-rewards](https://user-images.githubusercontent.com/39100405/51662694-f070a800-1fb4-11e9-931b-7657747e54bf.png)

The settings look like this:
![brave-settings](https://user-images.githubusercontent.com/39100405/51662781-388fca80-1fb5-11e9-8edb-8ee3c5eef6e6.png)

## Expected result:
The text should show up normally.

## Reproduces how often: 

Easily reproduced whenever I watch anything in the Brave Browser.

## Brave version (brave://version info)

Name | Value
-- | --
Brave | 0.58.21 Chromium: 71.0.3578.98 (Official Build) (64-bit)
Revision | 15234034d19b85dcd9a03b164ae89d04145d8368-refs/branch-heads/3578@{#897}
OS | Windows

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? 
It reproduces on beta builds.
### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
  - No.
- Is the issue reproducible on the latest version of Chrome? 
  - No.

### Additional Information

_None_

**Comment by srirambv** [comment]
Would be a good idea to check for driver updates on your machine.

**Comment by GeetaSarvadnya** [comment]
I have seen this issue on Windows 10 x64 once, after driver updates issue got resolved automatically.

**Comment by thepcwiz101** [comment]
i had this issue at one point but it stopped happening after a little while.

**Comment by rebron** [comment]
@onlyflix Would you be able to copy and send over the info you have in brave://gpu in the broken state so we can diagnose?

**Comment by onlyflix** [comment]
Of course, [here it is](https://gist.github.com/onlyflix/09048e7f8d3726ecff834e06ac96083a).

**Comment by iefremov** [comment]
@onlyflix Hi, can you please also share `chrome://gpu` info taken from Chrome on your machine?

**Comment by bsclifton** [comment]
@iefremov it should be shared above:
https://gist.github.com/onlyflix/09048e7f8d3726ecff834e06ac96083a

(copy/paste looks like it lost some of the formatting)

Is there anything in particular you're wanting to see?

**Comment by iefremov** [comment]
@bsclifton I assume that what is shared is the `brave://gpu` from Brave.
My idea is to compare this page snapshotted on Chrome versus  Brave `brave://gpu` to check any differences.

**Comment by bsclifton** [comment]
@iefremov ah - gotcha 😄 my bad! Would be great to compare

**Comment by onlyflix** [comment]
[This is the Chrome GPU log](https://gist.github.com/onlyflix/5dc69f33d5a02ed67561056283b3b257).

**Comment by iefremov** [comment]
Thanks @onlyflix 
Comparing two logs I can see that Brave uses discrete GPU, while Chrome stays on the on-chip one. Need to investigate why it happens and whats wrong with the discrete chip.

**Comment by GeetaSarvadnya** [comment]
1+ https://community.brave.com/t/distorted-web-content/51009

**Comment by iefremov** [comment]
Filed an issue discussed on Twitter: https://github.com/brave/brave-browser/issues/3969

**Comment by iefremov** [comment]
One more similar issue https://github.com/brave/brave-browser/issues/3471

In this one disabling the HW acceleration helps, and @kjozwiak reproduced the problem in Chrome on his VM.

**Comment by GeetaSarvadnya** [comment]
1+ https://community.brave.com/t/text-in-webpage-broken-or-hidden/65396/2

**Comment by bsclifton** [comment]
Closing as duplicate of https://github.com/brave/brave-browser/issues/3471
